app/gui/src/controller/searcher/input.rs

```diff
@@ -425,7 +426,11 @@ impl InsertContext<'_> {
                 .flatten()
                 .filter_map(|opr| ast::identifier::name(&opr.item.arg).map(ImString::new))
                 .collect_vec();
-            Some(name_segments)
+            if name_segments.is_empty() {
+                Some(name_segments)
+            } else {
+                None
+            }
```

Comment:
This looks like it should be if `!name_segments.is_empty()`?

Reply:
Sure, thank you!